src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -93,115 +96,136 @@ export class PropertyTemplateOutline extends Component {
     }
   }
 
-  handleComponentCase = (property) => {
+  /**
+   * returns jsx to render for passed property
+   * @private
+   */
+  propertyComponentJsx = (property) => {
     let rtReduxPath = Object.assign([], this.props.reduxPath)
-    let input, key, lookupConfigItem
+    let jsx, key, lookupConfigItem
```

Comment:
calling something we were going to be outputting "input" was confusing to me.  I think "jsx" covers the notion that it could be a React Component and/or html stuff -- "jsx" was the description of this sort of thing in the docs I was looking at

Reply:
👍